app/validation/validator.py

```diff
@@ -1491,11 +1520,19 @@ def _get_answers_with_parent_ids(self, json_to_validate):
         answers = {}
         for question, context in self._get_questions_with_context(json_to_validate):
             for answer in question.get('answers', []):
-                answers[answer['id']] = {
-                    'answer': answer,
-                    **context
-                }
+
+                if not answer['id'] in answers:
+                    answers[answer['id']] = {
+                        'answer': answer,
+                        **context
+                    }
+
                 for option in answer.get('options', []):
+                    answer_options = answers[answer['id']]['answer']['options']
+
+                    if answer_options and option['value'] not in [option['value'] for option in answer_options]:
+                        answer_options.append(option)
```

Comment:
These answer_options don't appear to be used for anything other than these three lines?

Reply:
Removed!